raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -108,14 +108,14 @@ def is_safe_to_wait(
     # A node may wait for a new balance proof while there are reveal_timeout
     # blocks left, at that block and onwards it is not safe to wait.
     if lock_timeout > reveal_timeout:
-        return True, None
+        return None
```

Comment:
This does not match the docstring and is also does not go well with the function name, because `is_safe_to_wait` will be falsy if it's safe and truthy if it's not. You've handled the case correctly below, so I guess this is just due to the early state of this PR.